authoring-new-actions.md

```diff
@@ -0,0 +1,130 @@
+# Process to publish GitHub Actions for Azure
+
+# Introduction
+
+[GitHub Actions](https://help.github.com/en/articles/about-github-actions) gives developers the flexibility to build an automated software development lifecycle workflow directly from their repositories. Since releasing Actions in November 2017, GitHub partners have contributed enormously to Marketplace, so teams can extend and automate workflows with their existing tools.
+
+Microsoft [announced](https://azure.microsoft.com/en-us/blog/github-actions-for-azure-is-now-generally-available/) the general availability of [GitHub Actions for Azure](https://github.com/Azure/actions) for creating workflows to package, release, and deploy apps to the cloud from GitHub. [GHActions\_vTeam](mailto:GHActions_vTeam@microsoft.com) has been formed to spearhead this effort of collaborating with various product teams at Microsoft to release these actions.
```

Comment:
Would we only allow contributions from MS only? Why not external contributor?

Reply:
We definitely need better name for our vTeam.

Reply:
You need to be a Microsoft employee to be able to create a repo under the Microsoft/Azure org. So, we can't open up authoring a new GitHub Action to the public; we can only allow contributions to the existing actions.

One alternative is to have a public repo where external contributors can raise a PR for proposing a new action; that way the `ace-team` can evaluate the scenario and get started with the authoring by accepting external contributions.

Reply:
Closed this with open-source team. Included only the URLs and no specifics about the steps prescribed on internal open-source portal.